Assets/_BForBoss/_Core/Scripts/EnergySystem/EnergySystemBehaviour.cs

```diff
@@ -179,6 +179,13 @@ private float MapToAccrueValue(EnergyAccruementType type, float multiplier)
         
         private float MapToExpendValue(EnergyExpenseType type, float multiplier)
         {
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+            if (UseDebugEnergyCost)
```

Comment:
We shouldn't be leaking Debug logic into the implementation itself. We want to keep it as separate as possible. 

If you want to make it so that the costs are all 0. Then create a new
`_energySystemConfigurationData = new GODMODECONFIG()`, and then set your own values in the partial. 

And then you can revert it after with the same `_energySystemConfigurationData = _energySystemConfiguration.MapToData();`. 

The point of setting everything as Data is that you can edit just the data/state, and it should work as you would like it to.